### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by George-Buzhenitsa**
- [DEMO LINK](https://George-Buzhenitsa.github.io/layout_stars/)
- [TEST REPORT LINK](https://George-Buzhenitsa.github.io/layout_stars/report/html_report/)

- [x] Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
- [X] Each BEM block has its own separate file
- [X] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [X] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/stars.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```
Comment (anastasiiavorobiova) [inline src/stars.css:17]:
```suggestion
}

.stars--1 :nth-child(-n + 1),
... ,
.stars--5 :nth-child(-n + 5) {
  background-image: url(./images/star-active.svg);
}
```

File: src/star-5.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```
Comment (anastasiiavorobiova) [inline src/star-5.css:5]:
```suggestion

```

File: src/star-1.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```
Comment (anastasiiavorobiova) [inline src/star-1.css:5]:
```suggestion

```

File: src/star-2.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```
Comment (anastasiiavorobiova) [inline src/star-2.css:5]:
```suggestion

```

File: src/star-3.css [hunk 5]
```diff
@@ -0,0 +1,5 @@
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```
Comment (anastasiiavorobiova) [inline src/star-3.css:5]:
```suggestion

```

File: src/star-4.css [hunk 6]
```diff
@@ -0,0 +1,5 @@
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```
Comment (anastasiiavorobiova) [inline src/star-4.css:5]:
```suggestion

```

File: src/stars.css (hunk 1 above)
Comment (anastasiiavorobiova) [reply src/stars.css:17]: Consider using commas to group selectors

File: src/style.css [hunk 7]
```diff
@@ -1 +1,6 @@
-/* add styles here */
+@import 'stars.css';
+@import 'star-1.css';
+@import 'star-2.css';
+@import 'star-3.css';
+@import 'star-4.css';
+@import 'star-5.css';
```
Comment (anastasiiavorobiova) [inline src/style.css:6]:
```suggestion

```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

File: src/stars.css (hunk 1 above)
Comment (George-Buzhenitsa) [reply src/stars.css:17]: as i understood i have to create only 2 files, one for general styles and another for active stars and just add all the selectors modification separated with coma? then import these 2 files in main file style.css

**Pull request closed without merging**